src/AioLogger.js

```diff
@@ -37,7 +48,7 @@ class AioLogger {
   setDefaults (moduleName, config) {
     config.level = config.level || DEFAULT_LEVEL
     config.provider = config.provider || DEFAULT_PROVIDER
-    config.logSourceAction = config.logSourceAction === undefined ? true : config.logSourceAction
+    config.logSourceAction = !!process.env.__OW_ACTION_NAME && config.logSourceAction !== false
```

Comment:
this could be a more clear for a maintainer of the code. Maybe add a comment, something like: 
```
// config.logSourceAction will only be TRUE if the both  __OW_ACTION_NAME env var is set and config.logSourceAction is TRUE

There really should have been tests originally for the 
default config of the Loggers - that the default config is what is advertised in the docs (test the combinations in the logic)

Right now the doc says `logSourceAction` defaults to true but it defaults to false if the __OW_ACTION_NAME is not set. I suspect the doc just needs to be clarified to "defaults to true if the __OW_ACTION_NAME is set, else it defaults to false. Set to false to disable this behaviour" or something to that effect
```

Reply:
ok thanks for the feedback, on it

Reply:
@shazron ready for new review :)